nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -474,6 +475,16 @@
             .allowableValues("true", "false")
             .build();
 
+    public static final PropertyDescriptor SUPPORT_HTTP2_PROTOCOL = new PropertyDescriptor.Builder()
+            .name("support-http2")
+            .description("Determines whether or not to support the HTTP 2 protocol version.")
```

Comment:
Recommend a slight wording change to use `HTTP/2` instead of `HTTP 2` in order to match official naming conventions.